sql/4_2_2-to-5_0_0_upgrade.sql

```diff
@@ -705,9 +705,17 @@ ALTER TABLE drugs
   ADD dispensable tinyint(1) NOT NULL DEFAULT 1 COMMENT '0 = pharmacy elsewhere, 1 = dispensed here';
 #EndIf
 
+#IfNotColumnType list_options notes TEXT
```

Comment:
with the error message of openemr or for long json of validation its should be type text